data_sync.R

```diff
@@ -3,17 +3,92 @@
 # of the data provided to us by Care Evolution(CE) in the input bucket.
 ##############
 ## Required functions and parameters
-source('awscli_utils.R')
-source('params.R')
+source('~/recover-s3-synindex/awscli_utils.R')
+source('~/recover-s3-synindex/params.R')
+library('synapser')
 
 #############
-# Sync S3 buckets 
+# Sync S3 ingress bucket to Local EC2 (First Sync )
+# using prod-creds (with 'read' only permissions) for accessing the ingress bucket 
 #############
-# <<values below are test values>>
-s3SyncBuckets(source_bucket = paste0('s3://', INGRESS_BUCKET,'/'),
-              destination_bucket = paste0('s3://', PRE_ETL_BUCKET,'/'))
+# synapser::synLogin(daemon_acc, daemon_acc_password) # login into Synapse
+synapser::synLogin()
+
+sts_token <- synapser::synGetStsStorageToken(entity = 'syn52293299', # sts enabled destination folder
```

Comment:
syn52293299 - Is the folder for MHP Input Data Internal attached to 	s3://recover-ingestion/

LOOKS GOOD